Gemfile.lock

```diff
@@ -87,7 +87,7 @@ GIT
 PATH
   remote: .
   specs:
-    packwerk (1.3.2)
+    packwerk (1.4.0)
```

Comment:
Let's do the version bump separately. I think I'll bump to 2.0, but we can discuss it separately, this PR can be merged without the bump.